src/ngAnimate/animateCss.js

```diff
@@ -903,10 +879,10 @@ var $AnimateCssProvider = ['$animateProvider', /** @this */ function($animatePro
           $$jqLite.addClass(element, activeClasses);
 
           if (flags.recalculateTimingStyles) {
-            fullClassName = node.getAttribute('class') + ' ' + preparationClasses;
-            cacheKey = gcsHashFn(node, fullClassName);
+            fullClassName = node.className + ' ' + preparationClasses;
```

Comment:
[According to MDN][1]:

> `className` can also be an instance of `SVGElement` if the element is an `SVGAnimatedString`. It is better to get/set the `className` of an element using `Element.getAttribute` and `Element.setAttribute` if you are dealing with SVG elements.

[1]: https://developer.mozilla.org/en-US/docs/Web/API/Element/className